store/migration/10002__add_vcs_github_com.sql

```diff
@@ -6,3 +6,10 @@ ALTER TABLE project_member ADD CONSTRAINT project_member_role_provider_check CHE
 
 ALTER TABLE vcs DROP CONSTRAINT vcs_type_check;
 ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST', 'GITHUB_COM'));
+
+ALTER TABLE sheet ADD source TEXT NOT NULL DEFAULT 'BYTEBASE' CHECK (source IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM'));
+ALTER TABLE sheet ADD type TEXT NOT NULL DEFAULT 'SQL' CHECK (type IN ('SQL'));
+ALTER TABLE sheet ADD payload JSONB NOT NULL DEFAULT '{}';
+
+ALTER TABLE repository ADD sheet_base_directory TEXT NOT NULL DEFAULT '';
```

Comment:
maybe we don't need sheet_base_directory.

We can specify that directory inside sheet_path_template

Reply:
Maybe we can rename the file to 10002__1.1.0 since we need to put these in 1.1.0.sql together anyway?

Reply:
> maybe we don't need sheet_base_directory.
> 
> We can specify that directory inside sheet_path_template

Agree, and users don't have to fill out one addition form.

Reply:
> Maybe we can rename the file to 10002__1.1.0 since we need to put these in 1.1.0.sql together anyway?

I prefer to name migration file by feature, which is a more common practice. e.g. https://gitlab.com/gitlab-org/gitlab-foss/-/blob/master/db/structure.sql

Reply:
“specify that directory inside sheet_path_template” +1.

Either way works as we will adopt semantic versioning, and the structure will be re-defined. For semantic versioning, it would make more sense to group by release versions - one file one migration version. For features, we can use comments to express them.

Reply:
> “specify that directory inside sheet_path_template” +1.
> 
> Either way works as we will adopt semantic versioning, and the structure will be re-defined. For semantic versioning, it would make more sense to group by release versions - one file one migration version. For features, we can use comments to express them.

Ok, should I rename this file in this PR?

Reply:
Yeah, maybe we can just do a rename if that's less confusing.

Reply:
I would still prefer a separate file to bump the migration number, is there a specific reason not doing this?

Reply:
I think either way works as we will do restructuring later for switching to the semantic version. This is the structure that I'm thinking of, and they do put different features into different files. We will do a split at that time, and I'm just thinking we don't need a perfect world at the moment, whatever works for now.

https://sourcegraph.com/github.com/temporalio/temporal@32410dcce0129ae1faa5bd57f5620dffd764404e/-/tree/schema/postgresql/v96/temporal/versioned/v1.5

Reply:
SGTM. As long as we will split them later.